### PR TITLE
refactor: update flow for not defined password

### PR DESCRIPTION
**Pull request by dudiq**

### CODE REVIEW

**Comment by dudiq** [comment]
@lamartire ready for review

File: src/components/screens/Authenticator/Authenticator.container.vue [hunk 1]
```diff
@@ -96,6 +96,10 @@ export default {
       const isEmailOtp = this.challengeType === CHALLENGE_TYPES.EMAIL_OTP;
 
       switch (true) {
+        case name === 'SignIn' && isEmailOtp && !this.isPasswordExist:
+          this.openRoute('SignUp');
```
Comment (lamartire) [inline src/components/screens/Authenticator/Authenticator.container.vue]: If we have created account, does it correct to force user for sign up? May be we should create another screen with separated functionality?

Comment (dudiq) [reply src/components/screens/Authenticator/Authenticator.container.vue]: if you doesn't have password, that means, that you are not registered in the system, that's why signUp will showed

Comment (dudiq) [reply src/components/screens/Authenticator/Authenticator.container.vue]: btw, this is the main goal

Comment (lamartire) [reply src/components/screens/Authenticator/Authenticator.container.vue]: It is very strange. So, ok

**Comment by lamartire** [comment]
@ifiri ready for approve

**Comment by ifiri** [comment]
@dudiq after signing in with empty github account, seeing this
image
Is this correct? I will be forced to create password.

**Comment by dudiq** [comment]
@ifiri this form will be removed
and this is not auth's form, this is vault's form.

**Comment by ifiri** [comment]
@dudiq then all is ok

**Comment by ifiri** [comment]
@dudiq after signup via google \ github I should not see Vault. I should see signup form instead.

**Comment by dudiq** [comment]
@ifiri 
- after signup via google \ github you should signed up to system
- you are testing `auth` with `vault` as i see and this dialog appears in `vault` only, not in `auth`.
try to check it in demo repo and you will not see this dialog

**Comment by ifiri** [comment]
@dudiq but I see the similar form in Auth as well.
image

**Comment by ifiri** [comment]
@kazurus approved

**Pull request merged**